compact/hcal/backward_endcap_flux.xml

```diff
@@ -0,0 +1,41 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2023 Wouter Deconinck, Leszek Kosarzewski -->
+
+<lccdd>
+  <define>
+    <documentation>
+      #### Material constants
+    </documentation>
+    <constant name="FluxEndcapN_zpos" value="FluxEndcapN_collar_thickness-FluxEndcapN_thickness"/>
+    <constant name="FluxEndcapN_zmin" value="BackwardServiceGap_zmax"/>
+
+  </define>
+
+
+  <!-- Define detector -->
+  <detectors>
+    <documentation>
+      ### Backwards (Negative Z) Endcap Hadronic Calorimeter
+    </documentation>
+    <detector id="FluxEndcapN_ID" name="FluxEndcapN" type="epic_EndcapFluxReturnN" vis="FluxEndcapNVis">
```

Comment:
Unless we store hits, we don't need an id for detectors.

Reply:
I removed the id. Somehow this strange spdx check failed. I don't think it's related.